fastlane/lanes/screenshots.rb

```diff
@@ -0,0 +1,230 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+#################################################
+# Constants
+#################################################
+
+# rubocop:disable Style/WordArray
+GLOTPRESS_TO_ASC_METADATA_LOCALE_CODES = [
+  ['ar', 'ar-SA'],
+  ['en-gb', 'en-US'],
+  ['en-gb', 'en-GB'],
```

Comment:
Disabled the rubocop rule because it would have felt a bit strange to write each entry in this map as `%w[ar ar-SA]`, given the real purpose is for this to be a mapping.

We will address this when we will reimplement the `gp_downloadmetadata` action during this sprint (see paaHJt-31O-p2#comment-5845) anyway; especially the new implementation will not only likely require a `Hash` instead of an `Array<Array<String>>`, but might also make use of [the future `LocaleHelper`](https://github.com/wordpress-mobile/release-toolkit/pull/296/files) at call site to provide the locales list in a more readable way. Which means this constant will be revisited soon anyway.